_students/ali-caudle.md

```diff
@@ -0,0 +1,3 @@
+---
+
+---
```

Comment:
@JessJodoin 

Because all of the requested naming conventions were lastname-firstname, and because I think we had already agreed to list students alphabetically by last name (I'll confirm this with branding to be sure), we're probably going to need to rename all of the student files 😭